pandas/tests/tools/test_to_datetime.py

```diff
@@ -2773,3 +2773,18 @@ def test_to_datetime_monotonic_increasing_index(cache):
     result = to_datetime(times.iloc[:, 0], cache=cache)
     expected = times.iloc[:, 0]
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "series_length",
+    range(40, start_caching_at + 5),
```

Comment:
Could you just test `[40, start_caching_at]`?

Reply:
I think its overkill to all values in  `range(40, start_caching_at + 5)`.

Could you update this to just test the values, [40, start_caching_at, start_caching_at + 1]

Reply:
Wouldn't it be better to go a few steps past `start_caching_at` so that it is a broader test?

Reply:
Okay sure, could also test `start_caching_at + 5` as well.

My main concern is a lot of redundant testing when it's known that the key bug occurs at the `start_caching_at` transition